grpc/response.go

```diff
@@ -27,10 +27,35 @@ func (r *Response) StatusCode() int {
 	return int(r.Status.Code())
 }
 
+// Label returns all the values associated with the given key, in the response metadata.
+// If the key does not exist, an empty slice will be returned.
+func (r *Response) Label(key string) []string {
+	return r.Metadata.Get(key)
+}
+
+// WithLabel appends the given value(s) to the key, in the response metadata.
+// If the key does not already exist, a new key will be created.
+// The modified response is returned.
+func (r *Response) WithLabel(key string, values ...string) fiber.Response {
+	r.Metadata.Append(key, values...)
+	return r
+}
+
+// WithLabels does the same thing as WithLabel but over a collection of key-values.
+func (r *Response) WithLabels(labels fiber.Labels) fiber.Response {
+	for _, key := range labels.Keys() {
+		values := labels.Label(key)
+		r.Metadata.Append(key, values...)
+	}
+	return r
+}
+
 func (r *Response) BackendName() string {
-	return strings.Join(r.Metadata.Get("backend"), ",")
+	return strings.Join(r.Label("backend"), ",")
```

Comment:
shouldn't this return what is set below with `r.Metadata.Set("backend", backendName)`, or that should have some changes?

Reply:
oh nvm me, i didnt catch it initially at lebels is merely setting in the metadata fields